tests/clone_test.c

```diff
@@ -28,7 +32,7 @@ clone_test (int converter)
 	SRC_STATE* src_state_cloned ;
 	SRC_DATA src_data, src_data_cloned ;
 
-	int error, frame, ch, index ;
+	int error, frame, ch, idx ;
```

Comment:
why the change?

Reply:
As mentioned in the commit message, it triggered -Wshadow and failed make distcheck on my setup.